src/components/ChatLog.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+
+const ChatLog = (props) => {
+    const entryComponents = props.entries.map((entry) => {
+        return (
+            <ChatEntry
+                key={`${entry.sender}-${entry.timeStamp}`}
+                id={entry.id}
+                sender={entry.sender}
+                body={entry.body}
+                timeStamp={entry.timeStamp}
+                liked={entry.liked} 
+                updateLike={props.updateLike} />
+        );
+    });
+
+    return (
+        <ul className="chat-log">{entryComponents}</ul>
```

Comment:
I didn't notice it until another student made the same mistake, but this ends up causing a slight violation of HTML expectations, as this `ul` tag has no `li` tags within it. Minor issue, though.